<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>209</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Stroke Color</string>
   </property>
   <widget class="QFrame" name="hue_frame">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>31</y>
      <width>40</width>
      <height>172</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="hue">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>6</y>
       <width>26</width>
       <height>160</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="hue_bg">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>6</y>
       <width>26</width>
       <height>160</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
border-radius: 5px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="hue_selector">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>160</y>
       <width>26</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #222;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="color_view">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>54</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* ALL CHANGES HERE WILL BE OVERWRITTEN */;
background-color: qlineargradient(x1:1, x2:0, stop:0 hsl(0.5%,100%,50%), stop:1 rgba(255, 255, 255, 255));

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="black_overlay">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 255));
border-radius: 4px;

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="selector">
      <property name="geometry">
       <rect>
        <x>-6</x>
        <y>144</y>
        <width>12</width>
        <height>12</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:none;
border: 1px solid white;
border-radius: 5px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="black_ring">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>1</y>
         <width>10</width>
         <height>10</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;
border: 1px solid black;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="color_vis">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>25</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
